api/v1beta1/condition_consts.go

```diff
@@ -51,4 +51,9 @@ const (
 	MissingBMHReason = "MissingBMH"
 	// Could not set the ProviderID on the target cluster's Node object.
 	SettingProviderIDOnNodeFailedReason = "SettingProviderIDOnNodeFailed"
+	// Metal3DataReadyCondition reports a summary of m3d status.
+	Metal3DataReadyCondition clusterv1.ConditionType = "Metal3DataReady"
+	// WaitingForMetal3DataReason used when waiting for metal3data
```

Comment:
can we please use consistent naming for the object in question: m3d vs metal3data

Reply:
fixed